mis_builder/static/src/components/mis_report_widget.xml

```diff
@@ -13,8 +13,7 @@
                         </div>
                         <div class="oe_mis_builder_cp_right_bottom">
                             <div class="oe_mis_builder_filter_buttons">
-                                <FilterMenu t-if="showSearchBar" />
-                                <DatePicker
+                                <DateTimePicker
```

Comment:
This looks likes this, with the full calendar being displayed:

![image](https://github.com/OCA/mis-builder/assets/692075/8cca86ac-693c-4634-8d8e-2e60331a5649)

It should be a date field with the calendar drop down.

Also I don't think we should use DateTime here, a Date is better.

Reply:
Fixed